vmf/scripts/mods/vmf/modules/ui/chat/chat_actions.lua

```diff
@@ -183,8 +183,24 @@ vmf:hook("ChatGui", "_update_input", function(func, self, input_service, menu_in
 
     -- ctrl + v
     if Keyboard.pressed(Keyboard.button_index("v")) and Keyboard.button(Keyboard.button_index("left ctrl")) == 1 then
-      -- remove CR characters
-      local new_chat_message = self.chat_message .. tostring(Clipboard.get()):gsub(string.char(0x0D), "")
+      local new_chat_message = self.chat_message
+      
+      -- remove carriage returns
+      local clipboard_data = tostring(Clipboard.get()):gsub(string.char(0x0D), "")
```

Comment:
I feel like ```:gsub("\r", "")``` would be a simpler alternative to using string.char

Reply:
That's part of the original code, I just moved it out of the way and clarified the comment.

I could still make it part of my changes, though.

Reply:
It would sanitize how cluttered that line is a little bit. The rest looks good though.